tests/acceptance/testutil/retry.go

```diff
@@ -62,6 +62,10 @@ func (r *R) Fail() {
 	r.failed = true
 }
 
+func (r *R) FailNow() {
```

Comment:
to satisfy `require.TestingT`, giving us more flexibility in reusing testing methods for that interface